robocorp-code/src/robocorp_code/inspector/web/_web_inspector.py

```diff
@@ -401,4 +403,4 @@ def inject_picker(self, reason: str, page=None) -> bool:
             return True
         except Exception:
             log.exception("Error injecting picker.")
-            return False
+            raise IOError("Error injecting picker.")
```

Comment:
This shouldn't raise an exception at this point. Is there a specific reason why you did this change?

Reply:
I haven't changed any PY files today. 
This must have been coming in from `master`.